spec/index.html

```diff
@@ -283,6 +283,14 @@ <h2>Introduction</h2>
   <section id="basic-concepts" class="informative">
     <h2>Basic Concepts</h2>
 
+    <p>
+      This document is based on YAML 1.2.2,
+      but YAML-LD is not tied to a specific version of YAML.
```

Comment:
Should be a bibliographic reference.

Reply:
Isn't [[YAML]] enough? If not, can you show me where to add it?

Reply:
> Isn't [[YAML]] enough? If not, can you show me where to add it?

That should do it. SpecRef links it to http://yaml.org/spec/1.2/spec.html.